psi4/driver/p4util/procutil.py

```diff
@@ -379,49 +390,92 @@ def reset_pe_options(pofm):
             if dlo['has_changed']:
                 core.set_local_option(module, lo, dlo['value'])
 
+    ## this is a more defensive version if defaults may have changed
+    # for go, dgo in pofm['GLOBALS'].items():
+    #    core.set_global_option(go, dgo['value'])
+    #    if not dgo['has_changed']:
+    #        core.revoke_global_option_changed(go)
+    # for module in _modules:
+    #    for lo, dlo in pofm[module].items():
+    #        core.set_local_option(module, lo, dlo['value'])
+    #        if not dlo['has_changed']:
+    #            core.revoke_local_option_changed(module, lo)
+
```

Comment:
Is there a reason to keep this?

Reply:
I'm willing to let it go in a future pass after ddd is stable, but I'd like to keep it for now.